blog/2024-05-01-gsoc-intern.md

```diff
@@ -0,0 +1,12 @@
+---
+author: Antonin Delpeuch
+title: "Zyad joins us as a Google Summer of Code contributor"
+---
+
+The OpenRefine project participates in the [Google Summer of Code](https://summerofcode.withgoogle.com/) program, which provides remote internships in software development on open source projects.
+We are happy to announce that for this year's cohort, we will be joined by [Zyad Taha](https://github.com/zyadtaha), who has applied to work on improvements to clustering.
+He will be mentored by [Antonin Delpeuch](https://github.com/wetneb).
+
+Please welcome Zyad in the community! Why don't you [introduce yourself to him](https://forum.openrefine.org/u/zyadtaha), explaining what you are doing in OpenRefine and why?
```

Comment:
```suggestion
Zyad already contributed several pull requests in the last week. However, this is the opportunity to welcome him into the community formally!  The Google Summer of Code internships will run from May 27 to August 26.
```

Reply:
The call to action (introducing and explaining) didn't match the link to Zyad profile. I replaced it with the internship date aligning with the format of our previous announcement.